armi/cases/tests/test_suiteBuilder.py

```diff
@@ -46,7 +46,13 @@ def __call__(self, cs, bp, geom):
 
 
 class TestLatinHyperCubeSuiteBuilder(unittest.TestCase):
-    """Class to test LatinHyperCubeSuiteBuilder."""
+    """
+    Class to test LatinHyperCubeSuiteBuilder.
+
+    .. test:: The run log has configurable verbosity.
```

Comment:
this test description doesn't seem right?

Reply:
debug, info, warning, error

We config the log system to only print out log statements of a minimum verbosity.

The phrasing seems right to me.   Thoughts?

Reply:
The requirement, `R_ARMI_CASE_MOD`, says
> The case package shall provide a generic mechanism to allow users to modify user inputs in a collection of cases.

which this test class tests. But it doesn't really have anything to do with the configurability of the runLog verbosity.

Reply:
Ah, I see. I'm just blind. One sec.

Reply:
Resolved.